### PR TITLE
Add About .NET section

### PR DESCRIPTION
**Pull request by richlander**
This is a start to the "About" section. I'd like it be grow to tell more of the .NET story over time, but I think it is big enough to merge now.

### CODE REVIEW

File: docs/about/products.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+- ".NET implementation" - Some combination of a runtime, framework and tools that can execute ".NET code" on which products are based.
+
+Product Categories
+------------------
+
+.NET products are available for each of the following product categories.
+
+### Desktop
+
+You can build desktop apps with for Windows and macOS.
```
Comment (svick) [inline docs/about/products.md]: Missing word: with → with .NET.

File: docs/about/products.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+.NET Core is used to build device, web, cloud and embedded/IoT apps. It is [open source](https://github.com/dotnet/core) and cross-platform, supporting Windows, macOS and Linux. [ASP.NET Core](http://docs.asp.net/) is the most popular workload for .NET Core. You can use it to build web apps and services, for on-premises and cloud deployment. You can also use .NET Core to build tools, utilities and cloud worker apps.
+
+- Learn about [.NET Core](../core)
+- Learn about [ASP.NET Core](http://docs.asp.net/)
+- [Download .NET Core](http://dot.net/core)
+
+The following are the main characteristics of .NET Core:
+
+**Cross-platform** - .NET Core supports three operating systems families: Linux, Windows and macOS. .NET Core apps are cross-platform by default. You need to build and publish a version per OS you want to target. The app host and the runtime are the primary OS-specific native dependencies. .NET Core libraries can run unmodified across supported OSes.
```
Comment (svick) [inline docs/about/products.md]: > You need to build and publish a version per OS you want to target.

Not if you're creating a portable application, which is the default.

File: docs/about/products.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+**Modular platform** - .NET Core is built with a modular design, enabling applications to include only the .NET Core libraries and dependencies that are needed. Each application makes its own .NET Core versioning choices, avoiding conflicts with shared components. This approach aligns with the trend of developing software using container technologies like Docker.
+
+.NET Framework
+--------------
+
+The .NET Framework is used to build apps for Windows and Windows Server. You can use it to build rich user interfaces with Windows Presentation Framework (WPF) and Windows Forms. You quickly build server apps with ASP.NET Web Forms, ASP.NET MVC and Windows Communication Framework (WCF). It is integrated into Visual Studio, which provides many rich designer experiences for building both client and server apps. It is the best choice for writing apps for Windows.
```
Comment (svick) [inline docs/about/products.md]: Saying that .NET Framework is integrated into VS kind of implies the other implementations are not. Is that intentional?

File: docs/about/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Multiple [.NET implementations and products](products.md) are available, based on open [.NET Standards](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) that specify the fundamentals of the platform. They are separately optimized for different application types (e.g. desktop, mobile, gaming, cloud) and support many chips (e.g. x86/x64, ARM) and operating systems (e.g. Windows, Linux, iOS, Android, macOS). Open source is also an important part of the .NET ecosystem, with multiple .NET implementations and many libraries available under OSI-approved licenses.
+
+- Learn about [.NET](../standard/)
+- Learn about [C#](../csharp/)
```
Comment (cartermp) [inline docs/about/index.md]: Since we also have "Learn F#" in the TOC, could we link to that as well?

**Comment by cartermp** [comment]
LGTM modulo minor comment

**Pull request merged**